src/Http/Controllers/WebhookController.php

```diff
@@ -53,7 +54,11 @@ public function __invoke(Request $request)
         WebhookReceived::dispatch($payload);
 
         if (method_exists($this, $method)) {
-            $this->{$method}($payload);
+            try {
+                $this->{$method}($payload);
+            } catch (InvalidPassthroughPayload $e) {
+                return new Response();
```

Comment:
Maybe we should say that we skipped the event, in all these cases, so things are clear from the paddle UI if someone wants to debug what's going with their hooks.

Reply:
Pushed a commit. Better?